powerauth-webflow/src/main/java/io/getlime/security/powerauth/app/webflow/configuration/WebMvcConfiguration.java

```diff
@@ -140,7 +141,11 @@ public ReloadableResourceBundleMessageSourceWithListing messageSource() {
 
     @Bean
     public Resource languageSettingSource() throws MalformedURLException {
-        return resourceLoader.getResource(configuration.getResourcesLocation() + "lang.json");
+        Resource resource = resourceLoader.getResource(configuration.getResourcesLocation() + "lang.json");
+        if (!resource.exists()) {
+            resource = new UrlResource("classpath:/static/resources/lang.json");
```

Comment:
May we log this branch, please? Something like
`logger.info("lang.json not found in {}", configuration.getResourcesLocation());`